build.sbt

```diff
@@ -85,7 +86,8 @@ lazy val MetricSettings = ColossusSettings
 
 lazy val ExamplesSettings = Seq(
   libraryDependencies ++= Seq(
-    "org.json4s" %% "json4s-jackson" % "3.5.3"
+    "org.json4s"     %% "json4s-jackson" % "3.5.3",
+    "ch.qos.logback" % "logback-classic" % "1.2.2"
```

Comment:
I don't think we should be adding logback as a dependency.  We don't want to cause issues with anyone using a different logging backend.  

I realize that realistically most people probably don't care and just want logging to work out of the box.  Maybe we can create a separate `colossus-logback` artifact or something that just adds the dependency and XML file.

Reply:
+1.  I think this will cause conflicts to users if they are not using logback, or are using different version of logback.  If we stick to SLF4J, it allows colossus to provide logging output, but allows the users to configure how and when they see it.

Reply:
Isn't this just adding it to the examples project? Therefore, isn't part of the colossus build?

Reply:
Ahhhh I missed that.  Well then everything looks fine to me. 💯

Reply:
That's for examples project only, so it can log correctly. It's not pulled into colossus.

Reply:
duh. Sorry